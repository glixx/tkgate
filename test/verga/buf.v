module top;
  reg a;
  wire x,y,z,i,j,k;

  initial
    $monitor("%4d: a=%b : x=%b  y=%b  z=%b : i=%b j=%b k=%b",$time,a,x,y,z,i,j,k);

  initial
    begin
	#10 a = 1;
	#10 a = 0;
	#10 a = 1;
	#10 a = 0;
    end
 
  buf #7 b(x,y,z,a);
  not #5 n(i,j,k,a);

endmodule
