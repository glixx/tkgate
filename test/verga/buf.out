   5: a=x : x=z  y=z  z=z : i=x j=x k=x
   7: a=x : x=x  y=x  z=x : i=x j=x k=x
  10: a=1 : x=x  y=x  z=x : i=x j=x k=x
  15: a=1 : x=x  y=x  z=x : i=0 j=0 k=0
  17: a=1 : x=1  y=1  z=1 : i=0 j=0 k=0
  20: a=0 : x=1  y=1  z=1 : i=0 j=0 k=0
  25: a=0 : x=1  y=1  z=1 : i=1 j=1 k=1
  27: a=0 : x=0  y=0  z=0 : i=1 j=1 k=1
  30: a=1 : x=0  y=0  z=0 : i=1 j=1 k=1
  35: a=1 : x=0  y=0  z=0 : i=0 j=0 k=0
  37: a=1 : x=1  y=1  z=1 : i=0 j=0 k=0
  40: a=0 : x=1  y=1  z=1 : i=0 j=0 k=0
  45: a=0 : x=1  y=1  z=1 : i=1 j=1 k=1
  47: a=0 : x=0  y=0  z=0 : i=1 j=1 k=1
